File: problem/Min_K_Scores/minimumDifference.c
int cmp(const void *a, const void *b)
{
    return (*(int *)a - *(int *)b);
}

int minimumDifference(int *nums, int numsSize, int k)
{
    qsort(nums, numsSize, sizeof(int), cmp);
    int ans = nums[k - 1] - nums[0];
    for (int i = 1; i + k <= numsSize; ++i)
    {
        int diff = nums[i + k - 1] - nums[i];
        if (diff < ans)
        {
            ans = diff;
        }
    }
    return ans;
}

File: problem/Min_K_Scores/minimumDifference.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minimumDifference = function(nums, k) {
    nums.sort((a, b) => a - b);
    let ans = nums[k - 1] - nums[0];
    for (let i = 1; i + k <= nums.length; i++) {
        ans = Math.min(ans, nums[i + k - 1] - nums[i]);
    }
    return ans;
};